documentation/advanced/tutorial-i18n-localization.asciidoc

```diff
@@ -169,3 +169,139 @@ public class MyLocale extends Div {
     }
 }
 ----
+
+=== Supporting Right-to-Left mode in the application
```

Comment:
Why a third level heading? To me it seems a second level heading would work fine.

The writing style for our docs uses Title Case. Should fix the one heading above to follow that as well.

```suggestion
== Supporting Right-to-Left Mode
```

Reply:
I guess I should fix all the titles on this page to follow this style, then.

Reply:
Changed titles.